.github/workflows/check_atlantis.yaml

```diff
@@ -21,12 +7,12 @@ jobs:
   check_docs_up_to_date:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Check Docs for all modules
         uses: terraform-docs/gh-actions@v1.2.0
         with:
           find-dir: .
           config-file: .terraform-docs.yml
-          output-file: README.md

```

Comment:
does it mean the default value is README.md?

Reply:
yes